Add a missing #include directive to avoid an implicit function declaration.

diff -ur squidGuard-1.4.orig/src/sgDiv.c.in squidGuard-1.4/src/sgDiv.c.in
--- squidGuard-1.4.orig/src/sgDiv.c.in	2022-11-26 16:54:31.560596774 +0100
+++ squidGuard-1.4/src/sgDiv.c.in	2022-11-26 17:04:16.724471141 +0100
@@ -18,6 +18,7 @@
 
 #include "sg.h"
 #include "sgEx.h"
+#include "HTEscape.h"
 
 /* #define METEST 8; */
 
